ou_dedetai/installer.py

```diff
@@ -349,8 +347,27 @@ def create_wine_appimage_symlinks(app: App):
         p.symlink_to(f"./{app.conf.wine_appimage_link_file_name}")
 
 
-def create_desktop_file(name, contents):
-    launcher_path = Path(f"~/.local/share/applications/{name}").expanduser()
+def create_desktop_file(
+    filename: str,
+    app_name: str,
+    exec_cmd: str,
+    icon_path: str,
+    wm_class: str,
+):
+    contents = f"""[Desktop Entry]
+Name={app_name}
+GenericName=FaithLife Wine App Installer
+Comment=Manages FaithLife Bible Software via Wine
+Exec={exec_cmd}
+Icon={icon_path}
+Terminal=false
+Type=Application
+StartupWMClass={wm_class}
+Categories=Education;
+Keywords=Logos;Verbum;Bible;Control;
+"""
+    xdg_data_home = Path(os.getenv('XDG_DATA_HOME', Path.home()))
```

Comment:
Shouldn't this default to ~/.local/share?

Reply:
oh, yes. Good catch.